### ISSUE TITLE
URL for HuggingFace do not work on Windows ("/" are replaced by "%5C")

### ISSUE THREAD

**Issue by clmpt** [pyannote/pyannote-audio#633]
It triggers a HTTPError.

I think I have found the reason of this problem:
In [pyannote/audio/core/inference.py#L98 ](https://github.com/pyannote/pyannote-audio/blob/e3347fab9abd3842d20ef8f31b0b0ff9cca44228/pyannote/audio/core/inference.py#L98), `Path` from `pathlib` converts the slashes in the URL into backslashes, because paths in windows are with backslashes.

I got the following error log when trying to execute the code from this [notebook](https://huggingface.co/hbredin/VoiceActivityDetection-PyanNet-DIHARD):

```
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
<ipython-input-28-f81de430577a> in <module>
      1 MODEL = "hbredin/VoiceActivityDetection-PyanNet-DIHARD"
      2 from pyannote.audio import Pipeline
----> 3 vad = Pipeline.from_pretrained(MODEL, device="cuda")
      4 speech_regions = vad("audio.wav")
      5 for (start_time, end_time) in speech_regions.get_timeline():

E:\Logiciels\Anaconda\envs\Stage - PyAnnot\lib\site-packages\pyannote\audio\core\pipeline.py in from_pretrained(cls, checkpoint_path, hparams_file, device, batch_size, use_auth_token, progress_hook)
    141                             params["progress_hook"] = key
    142 
--> 143                     preprocessors[key] = Klass(**params)
    144 
    145                     continue

E:\Logiciels\Anaconda\envs\Stage - PyAnnot\lib\site-packages\pyannote\audio\core\inference.py in __init__(self, model, window, skip_aggregation, device, duration, step, batch_size, progress_hook, use_auth_token)
     97             model
     98             if isinstance(model, Model)
---> 99             else Model.from_pretrained(
    100                 Path(model),
    101                 map_location=device,

E:\Logiciels\Anaconda\envs\Stage - PyAnnot\lib\site-packages\pyannote\audio\core\model.py in from_pretrained(cls, checkpoint, map_location, hparams_file, strict, task, use_auth_token, **kwargs)
    774             )
    775 
--> 776             path_for_pl = cached_download(
    777                 url=url,
    778                 library_name="pyannote",

E:\Logiciels\Anaconda\envs\Stage - PyAnnot\lib\site-packages\huggingface_hub\file_download.py in cached_download(url, library_name, library_version, cache_dir, user_agent, force_download, proxies, etag_timeout, resume_download, use_auth_token, local_files_only)
    262                 timeout=etag_timeout,
    263             )
--> 264             r.raise_for_status()
    265             etag = r.headers.get("X-Linked-Etag") or r.headers.get("ETag")
    266             # We favor a custom header indicating the etag of the linked resource, and

E:\Logiciels\Anaconda\envs\Stage - PyAnnot\lib\site-packages\requests\models.py in raise_for_status(self)
    941 
    942         if http_error_msg:
--> 943             raise HTTPError(http_error_msg, response=self)
    944 
    945     def close(self):

HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/hbredin%5CVoiceActivityDetection-PyanNet-DIHARD/resolve/main/pytorch_model.bin
```

**Comment by hbredin** [comment]
Good catch. Thanks. Would you mind contributing a fix with a pull request? 

FYI, Windows is not officially supported -- though I'd be happy to receive feedback about how the codebase behave on Windows...

**Comment by clmpt** [comment]
I fixed this issue.

However, I'm still trying to make `pyannote` work on Windows as `torch` does not recognize my GPU because of the Unix way of installing the library. (Is it possible to make `pyannote` work with CPU-only configurations ?)

I added `platfrom_system` conditions to the requirements, yet I'm unable to build the wheel (```error in pyannote.audio setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Parse error at "'-f https'": Expected stringEnd```). I will try to find a solution for that...
